packages/element/src/react.js

```diff
@@ -109,8 +111,24 @@ export { isValidElement };
  */
 export { StrictMode };
 
+/**
+ * The React.lazy function lets you render a dynamic import as a regular component.
+ *
+ * @param {function} loadFunction A function that must call a dynamic import(). This must return a Promise which
+ *                                resolves to a module with a default export containing a React component.
+ */
+export { lazy };
+
+/**
+ * Component to do code-splitting by wrapping a dynamic import.
+ */
+export { Suspense };
+
 /**
  * Make React Hooks available
+ *
+ * @param {function} loadFunction A function that must call a dynamic import(). This must return a Promise which
```

Comment:
I don't think this was meant to be included here?

Reply:
Right, fixed. Not sure why there are unrelated docs changes even after merging master and running `npm run docs:build` - am I missing a step?

Reply:
>running npm run docs:build - am I missing a step?

Can you try it once more? The changes I would have expected with this erroneous DocBlock, but I expect should be largely cleared up if run in the current state of the branch. Would also explain the failing build.

Reply:
grabbed readme from master, then re-ran - changes in https://github.com/WordPress/gutenberg/pull/14412/commits/1466b25e3f4d31551290b916cb294868b10486f1.